FWCore/MessageService/interface/ELstatistics.h

```diff
@@ -34,98 +33,89 @@
+      ELstatistics(std::ostream& osp);
+      ELstatistics(int spaceLimit);
+      ELstatistics(int spaceLimit, std::ostream& osp);
+      ELstatistics(const ELstatistics& orig);
+      ~ELstatistics() override;
 
-public:
-  // -----  constructor/destructor:
-  ELstatistics();
-  ELstatistics( std::ostream & osp );
-  ELstatistics( int spaceLimit );
-  ELstatistics( int spaceLimit, std::ostream & osp );
-  ELstatistics( const ELstatistics & orig );
-  ~ELstatistics() override;
+      // -----  Methods invoked by the ELadministrator:
+      //
+    public:
+      // Used by attach() to put the destination on the ELadministrators list
+      //-| There is a note in Design Notes about semantics
+      //-| of copying a destination onto the list:  ofstream
+      //-| ownership is passed to the new copy.
 
-  // -----  Methods invoked by the ELadministrator:
-  //
-public:
-    // Used by attach() to put the destination on the ELadministrators list
-		//-| There is a note in Design Notes about semantics
-		//-| of copying a destination onto the list:  ofstream
-		//-| ownership is passed to the new copy.
+      bool log(const edm::ErrorObj& msg) override;
 
-  bool log( const edm::ErrorObj & msg ) override;
+      // output( const ELstring & item, const ELseverityLevel & sev )
+      // from base class
 
-  // output( const ELstring & item, const ELseverityLevel & sev )
-  // from base class
+      // ----- Methods invoked by the MessageLoggerScribe, bypassing destControl
+      //
+    public:
+      static void noteGroupedCategory(std::string const& cat);  // 8/16/07 mf
 
-  // ----- Methods invoked by the MessageLoggerScribe, bypassing destControl
-  //
-public:
-  static void noteGroupedCategory(std::string const & cat);  // 8/16/07 mf 
+      void summary(unsigned long overfullWaitCount);
+      void noTerminationSummary();
+      void summaryForJobReport(std::map<std::string, double>& sm);
+      void wipe() override;
 
+    protected:
+      void clearSummary();
 
-  void summary( unsigned long overfullWaitCount );
-  void noTerminationSummary();
-  void summaryForJobReport (std::map<std::string, double> & sm);
-  void wipe() override;
+      void zero() override;
 
-protected:
-  void clearSummary();
+      std::map<ELextendedID, StatsCount> statisticsMap() const;
 
-  void zero() override;
+      // summarization( const ELstring & sumLines, const ELstring & sumLines )
+      // from base class
 
+    protected:
+      int tableLimit;
+      ELmap_stats stats;
+      bool updatedStats;
+      std::ostream& termStream;
 
-  std::map<ELextendedID,StatsCount> statisticsMap() const;
+      bool printAtTermination;
 
-  
-  // summarization( const ELstring & sumLines, const ELstring & sumLines )
-  // from base class
+      CMS_THREAD_SAFE static std::set<std::string> groupedCategories;  // 8/16/07 mf
+      static ELstring formSummary(ELmap_stats& stats);                 // 8/16/07 mf
 
-protected:
-  int            tableLimit;
```

Comment:
I was never a fan of the two column approach of declaring member variables anyway.